changes/tasks.py

```diff
@@ -251,6 +251,7 @@ def pmtct_loss_switch(self, change):
 
         if switched is True:
             self.l.info("Completed PMTCT switch to loss")
+            push_momconnect_babyloss_to_jembi.delay(str(change.pk))
```

Comment:
I think this needs to be added to loss switches from other services as well. So here https://github.com/praekeltfoundation/ndoh-hub/blob/32f384bf43a4089e953be84a8e4c8338ebcc93da/changes/tasks.py#L314